### PR TITLE
Add Distributed Instructlab Training on RHOAI doc

### PR DESCRIPTION
**Pull request by abhijeet-dhumal**
## Description

Added instructions to run Distributed Instructlab Training on RHOAI.
Migrating this PR with additional changes : https://github.com/opendatahub-io/distributed-workloads/pull/268

## How Has This Been Tested?

image (183)

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:9]: Is the number of nodes a pre-requisite? My understanding is what matters is the number of GPUs.

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:93]: Should the URL be changed to point to that repo?

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:24]:
```suggestion
* Optional Step 3 - Setup NFS provisioner
```

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:149]: We can remove "Option 2".

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:144]: We can add that it's only needed when the cluster doesn't have a provisioner with ReadWriteMany capability.

File: standalone/README.md
Comment (sutaakar) [inline standalone/README.md:351]:
```suggestion
If you want to run the `standalone.py` script with Red Hat OpenShift AI you can try out this [experimental install guide](instructlab-rhoai.md).
```

File: standalone/instructlab-rhoai.md
Comment (abhijeet-dhumal) [reply standalone/instructlab-rhoai.md:9]: I guess the related config info is taken from here : https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_ai/1.1/html/getting_started/hardware_requirements_rhelai#bare-metal
But need to reframe the sentence a bit

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:24]: nit: "Create nfs server" -> "Setup NFS StorageClass"

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:175]: This should be changed to point to phase one branch of that repository.

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:162]: I'd suggest to call that part "Step 4 - Run InstructLab distributed training" and do not number the included steps to keep the logical structure of the process simple.

**Review by astefanutti — COMMENTED** [review]
I've just left a couple of comments, I think it should be good to go after that.

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:159]: I think running the `oc new-project nfs` command is needed first.

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:157]: nit: "a" -> "an"

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:159]: I suggest to add a quick not about checking the root PVC that'll be created and the requested storage.

File: standalone/instructlab-rhoai.md
Comment (astefanutti) [inline standalone/instructlab-rhoai.md:16]: Maybe add the `oc` binary should be installed locally.

Comment (abhijeet-dhumal) [reply standalone/instructlab-rhoai.md:16]: Is this draft seems correct? 
```
The `oc` command line tool binary installed locally to create and manage kubernetes resources.
```

**Comment by abhijeet-dhumal** [comment]
Thank you @astefanutti @sutaakar for reviewing this PR, 
I'm grateful for your detailed reviews !! :rocket:

**Comment by astefanutti** [comment]
@sutaakar could you please give it a LGTM if / once it does?

@abhijeet-dhumal could you please back-port it in the `phase-1` branch once it's merged?

**Review by sutaakar — APPROVED** [review]
Thanks

**Comment by astefanutti** [comment]
@abhijeet-dhumal could you please create a back-port PR?

**Comment by abhijeet-dhumal** [comment]
> @abhijeet-dhumal could you please create a back-port PR?

@astefanutti @sutaakar Please review : https://github.com/opendatahub-io/ilab-on-ocp/pull/177

**Pull request merged**
